org.eclipse.xtext/src/org/eclipse/xtext/nodemodel/impl/InternalNodeModelUtils.java

```diff
@@ -58,7 +58,7 @@ protected static LineAndColumn getLineAndColumn(INode anyNode, int documentOffse
 	 */
 	protected static LineAndColumn getLineAndColumn(String text, int[] lineBreaks, int offset) {
 		if (offset > text.length() || offset < 0) {
-			throw new IndexOutOfBoundsException();
+			throw new IndexOutOfBoundsException("Offset (" + offset + ") is longer than the text's length (" + text.length() + ")");
```

Comment:
might miss negative offsets

Reply:
Maybe use `com.google.common.base.Preconditions.checkElementIndex`?

Reply:
Well in fact it would need to be checkPositionIndex.

Reply:
Done